Sources/Sentry/include/SentryLog.h

```diff
@@ -16,15 +16,35 @@ SENTRY_NO_INIT
 NS_ASSUME_NONNULL_END
 
 #define SENTRY_LOG_DEBUG(...)                                                                      \
-    [SentryLog logWithMessage:[NSString stringWithFormat:__VA_ARGS__] andLevel:kSentryLevelDebug]
+    [SentryLog logWithMessage:[NSString stringWithFormat:@"[%@:%d] %@",                            \
+                                        [[[NSString stringWithUTF8String:__FILE__]                 \
+                                            lastPathComponent] stringByDeletingPathExtension],     \
+                                        __LINE__, [NSString stringWithFormat:__VA_ARGS__]]         \
+                     andLevel:kSentryLevelDebug]
```

Comment:
The change that automatically adds the log source to all logs using these macros.